wizzard_dialog.py

```diff
@@ -0,0 +1,197 @@
+
+WIDGET, BASE = uic.loadUiType(os.path.join(this_dir, 'wizzard_dialog.ui'))
+
+
+class WizzardDialog(BASE, WIDGET):
+    def __init__(self, conn, parent=None):
+        BASE.__init__(self, parent)
+        self.setupUi(self)
+
+        self.conn = conn
+
+        self.table_model = QStandardItemModel()
+        self.tableView.setModel(self.table_model)
+
+        for schema_oid, schema_name in trigger_dialog.get_schemas(conn):
+            self.cboSourceSchema.addItem(schema_name)
+            self.cboTargetSchema.addItem(schema_name)
+
+        self.cboSourceSchema.currentIndexChanged.connect(self.populate_tables)
+        self.cboTargetSchema.currentIndexChanged.connect(self.populate_tables)
+
+        self.tableFld.textChanged.connect(self.populate_tables)
+        self.cboTablesOpt.currentIndexChanged.connect(self.populate_tables)
+
+        self.attrFld.textChanged.connect(self.populate_tables)
+        self.cboFieldsOpt.currentIndexChanged.connect(self.populate_tables)
+
+        self.layers = trigger_dialog.get_spatial_tables(conn)
+        self.populate_tables()
+
+    # TODO support for regex??
+    def filter_prefix_suffix(self, list, prefix, suffix):
+        result = []
+        ix = 0
+        for item in list:
+            if prefix and item.startswith(prefix):
+                item = item.replace(prefix, "", 1)
+            if suffix and item.endswith(suffix):
+                item = "".join(item.rsplit(suffix, 1))
+
+            result.append((item, ix))
+            ix += 1
+        return result
+
+    def update_table_model(self, pairs):
+        self.table_model.clear()
+        self.table_model.setHorizontalHeaderLabels(["Source", "Target"])
+        ix = -1
+
+        for source, target in pairs:
+            ix += 1
+            item_0 = QStandardItem(source)
+            item_1 = QStandardItem(target)
+            for i in [item_0, item_1]:
+                i.setEditable(False)
+            self.table_model.appendRow([item_0, item_1])
+
+        self.tableView.resizeColumnToContents(1)
+
+    def populate_tables(self):
+        tabs1 = self.get_tables(self.cboSourceSchema.currentText())
+        tabs2 = self.get_tables(self.cboTargetSchema.currentText())
+
+        prefixSource = self.tableFld.text() if self.cboTablesOpt.currentIndex() == 1 else ""
+        suffixSource = self.tableFld.text() if self.cboTablesOpt.currentIndex() == 2 else ""
+        prefixTarget = self.tableFld.text() if self.cboTablesOpt.currentIndex() == 3 else ""
+        suffixTarget = self.tableFld.text() if self.cboTablesOpt.currentIndex() == 4 else ""
+
+        tab11, tab22 = self.get_similar(tabs1, tabs2, prefixSource, suffixSource, prefixTarget, suffixTarget)
+
+        source = [elem for (elem, schema) in tab11]
+        target = [elem for (elem, schema) in tab22]
+
+        pairs = [(source[ix], target[ix]) for ix in range(len(source))]
+        self.update_table_model(pairs)
+        self.populate_attr(pairs)
```

Comment:
It would be also useful to expand the tree view here so it is possible to see the matching attributes